client/views/chan.tpl

```diff
@@ -1,7 +1,16 @@
 {{#each channels}}
 <div data-id="{{id}}" data-target="#chan-{{id}}" data-title="{{name}}" class="chan {{type}} chan-{{slugify name}}">
 	<span class="badge{{#if highlight}} highlight{{/if}}">{{#if unread}}{{roundBadgeNumber unread}}{{/if}}</span>
-	<button class="close" aria-label="Close"></button>
+		{{#equal type "channel"}}
+			<span class="close-tooltip tooltipped tooltipped-w" aria-label="Leave">
+				<button class="leave" aria-label="Leave"></button>
```

Comment:
Leave this class as `close`, less risk to break some consistency/action IMO. Also, you let the tooltip class as `close-tooltip` for both, which kinda shows what I mean ahah.

Reply:
That was on purpose, as to avoid adding a CSS line and I couldn't think of a more generic classname, but yeah point taken.

Reply:
Just use `.close` for both and it'll be fine.